src/models/aspera-sdk.model.ts

```diff
@@ -119,13 +121,46 @@ export class ActivityTracking {
       return;
     }
 
+    this.lastWebSocketEvent = webSocketEvent;
```

Comment:
Sorry, kind of confused - aren't `this.lastWebSocketEvent` and `this.lastNotifiedWebSocketEvent` the same thing?

Can we remove `this.lastNotifiedWebSocketEvent` and just use the existing `this.lastWebSocketEvent`?

Reply:
No, they are not the same thing.

I added the new `lastNotifiedWebSocketEvent` since we still need to know what was the last real `WebSocketEvent` received to notify it to the registered callbacks when the client status changes.

Unless we always assume that the websocket state is always `CLOSED` when the application is "not running".